### PR TITLE
[Kernel CI] Change some aur packages to official packages

### PR DESCRIPTION
**Pull request by Nevuly**
Error Logs:
:: Synchronizing package databases...
 core downloading...
 extra downloading...
 archlinuxcn downloading...
error: target not found: llvm-git
error: target not found: llvm-libs-git
error: target not found: clang-git
error: target not found: lld-git

 * llvm-git, llvm-libs-git, clang-git, lld-git is aur packages, It can't install with pacman.
 * They support at official packages, So change it.

### CODE REVIEW

**Comment by Nevuly** [comment]
Today, I checked archlinuxcn, It removed in repo because they triggered to orphan packages.

https://github.com/archlinuxcn/repo/commit/960f21835fe2bc68638d44e100b4e018e9b060b0

**Review by taalojarvi — CHANGES_REQUESTED** [review]
LTS builds do not support ccache at the moment

File: .github/workflows/build-lts.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
     - name: Install dependencies
       id: dep
       run: |
-        pacman -Syu --noconfirm pahole xmlto inetutils bc cpio jq llvm llvm-libs clang lld
+        pacman -Syu --noconfirm pahole xmlto inetutils bc cpio jq llvm llvm-libs clang lld ccache
```
[the remark this answers is not in the document]
Comment (Nevuly) [reply .github/workflows/build-lts.yml:45]: I'll update PR ASAP.

[the remark this answers is not in the document]
Comment (Nevuly) [reply .github/workflows/build-lts.yml]: Reverted.

**Review by taalojarvi — APPROVED** [review]

**Comment by Nevuly** [comment]
I close this PR because It has been pending more than 2 weeks.

If this PR is necessary, please reopen.

Thanks.

**Pull request closed without merging**